Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -477,6 +478,11 @@ def prepare(self,mReader, directories, mode='init'):
                     chainDict['RequestString']='RV'+chainDict['CMSSWVersion']+s[1].split('+')[0]
                     if processStrPrefix or thisLabel:
                         chainDict['RequestString']+='_'+processStrPrefix+thisLabel
+                    #check candidate WF name
+                    self.candidateWFName = self.user+'_'+chainDict['RequestString']
+                    if (len(self.candidateWFName)>81):
```

Comment:
magic number 81 should be named constant

Reply:
Done, define a constant MAXWORKFLOWLENGTH at the beginning of the file.